src/main/java/appeng/fluids/container/ContainerFluidLevelEmitter.java

```diff
@@ -0,0 +1,97 @@
+
+package appeng.fluids.container;
+
+
+import net.minecraft.client.gui.GuiTextField;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+import appeng.api.config.RedstoneMode;
+import appeng.api.config.SecurityPermissions;
+import appeng.api.config.Settings;
+import appeng.container.guisync.GuiSync;
+import appeng.fluids.parts.PartFluidLevelEmitter;
+import appeng.fluids.util.IAEFluidTank;
+import appeng.util.Platform;
+
+
+public class ContainerFluidLevelEmitter extends ContainerFluidConfigurable
+{
+	private final PartFluidLevelEmitter lvlEmitter;
+
+	@SideOnly( Side.CLIENT )
+	private GuiTextField textField;
+	@GuiSync( 3 )
+	public long EmitterValue = -1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "EmitterValue" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00116)